.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -90,4 +90,6 @@ will share this change with the people who need to know about it._
 
 ## [optional] What gif best describes this PR or how it makes you feel?
 
-![alt_text](gif_link)
+<!-- ![gif-name](link)  or -->
+<!-- <img src="gif-link" width="120" height="auto"/> -->
```

Comment:
@KirtanSoni do you think perhaps we should leave it as `![alt_text](gif_link)`? The reason being is that if folks want to add a gif, then they don't need to uncomment the comments that you've added but can just replace `gif_link` with the actual link.

Reply:
Ah, I just posed the same question and asked for your input, @Ridhwana! 😄